GameServerLib/GameObjects/AttackableUnits/AttackableUnit.cs

```diff
@@ -457,8 +473,11 @@ public void RemoveStatModifier(IStatsModifier statModifier)
                 attackerId = (int)_game.PlayerManager.GetClientInfoByChampion((IChampion)attackerMinion.Owner).PlayerId;
             }
 
-            _game.PacketNotifier.NotifyUnitApplyDamage(attacker, this, damage, type, damageText,
-                _game.Config.IsDamageTextGlobal, attackerId, targetId);
+            if (attacker.Team != Team)
```

Comment:
Do we have a flag for that `IsTargetableToTeam` or similar? Or is it always true that you cannot attack your own team players?

Reply:
This is just for the damage text. Minions often suicide using TakeDamage, so this prevents the text from appearing.